packages/EasyCore/src/Search/ElasticsearchSearchService.php

```diff
@@ -61,4 +62,26 @@ public function search(
             'body' => $body,
         ]);
     }
+
+    /**
+     * @param mixed[] $query
+     *
+     * @return mixed[]
+     */
+    private function fixMatchAllRecursive(array $query): array
```

Comment:
Let's rename to `replaceEmptyArrayWithEmptyObjectInMatchAllRecursively`

Reply:
Done.